lib/xrandr.rb

```diff
@@ -36,6 +36,16 @@ def apply!
       Kernel.system(command)
       initialize
     end
+
+    def connected_outputs
+      outputs.select { |output| output.connected }
+    end
+
+    def disconnected_outputs
+      outputs.reject { |output| output.connected }
+    end
+
+
```

Comment:
:policeman:  THE CODE STYLE COP says:  _Would you please remove this extra white line?_